### PR TITLE
Add .timer command.

### PR DESCRIPTION
**Pull request by tzzed**
This is a proposal for #443.

### CODE REVIEW

File: cmd/genji/shell/shell.go [hunk 1]
```diff
@@ -189,12 +195,34 @@ func (sh *Shell) runExecutor(ctx context.Context, promptExecCh chan string) erro
...
+			if showTime && !strings.HasPrefix(input, ".") && strings.HasSuffix(input, ";") {
+				displayTime(start)
 			}
 		case promptExecCh <- "":
 		}
 	}
 }
 
+// displayTime shows execution time of a query.
+func displayTime(start time.Time) {
```
Comment (asdine) [inline cmd/genji/shell/shell.go:211]: How about using the default display of the [time.Duration](https://pkg.go.dev/time#Duration.String) type?

**Review by asdine — COMMENTED** [review]
Thanks @tzzed ! Just one comment

File: cmd/genji/shell/shell.go (hunk 1 above)
Comment (asdine) [reply cmd/genji/shell/shell.go:211]: I think we can just use the time.Duration output yeat

Comment (tzzed) [reply cmd/genji/shell/shell.go:211]: Simple and efficient!

**Review by asdine — APPROVED** [review]
✨

**Pull request merged**